src/utils.js

```diff
@@ -40,6 +41,21 @@ export function chainWith(createDependentObservable) {
   });
 }
 
+/**
+ * Custom rxjs operator to emit an initial value even if each observable hasn't emmited a value yet .
+ * Usage:
+ ```js
+  combineLatestImmediate(obs1, obs2);
+  ```
+ *
+ * @param {Observable} obs1 Observable that emits values depending on where it is used
+ * @param {Observable} obs2  Observable that emits values depending on where it is used
```

Comment:
actually, this should work with any number of input observables, just like combineLatest does:
```js
combineLatestImmediate(...observables) {
   combineLatest(observables.map(obs => obs.pipe(startWith(undefined)));
}